internal/k8s/client.go

```diff
@@ -108,11 +107,6 @@ func NewK8sClient(
 	restConfig *rest.Config,
 	pf PortForwarder) K8sClient {
 
-	// TODO(nick): I'm not happy about the way that pkg/browser uses global writers.
-	writer := logger.Get(ctx).Writer(logger.DebugLvl)
```

Comment:
why did you remove this? this is pretty important

Reply:
Yeah I realized that and added it back.